src/App.jsx

```diff
@@ -0,0 +1,15 @@
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import Article from "./pages/Article";
+import Home from "./pages/Home";
+function App() {
```

Comment:
Try to follow the same pattern of exporting components

```suggestion
export default function App() {
```